### DISCUSSION TITLE
Simple humble question from a 100% noob...any kind words of advice would be greatly appreciated!

### DISCUSSION THREAD

**Discussion by baysidemeef** [community/community#56007] [Programming Help] [answered]
### Body

I'm just starting to get into computer science. I know very little. Last year I dabbled with Arduino then I completed CS50 (except the final project). Now just as a bored test project, I created a very very simple rudimentary HTML app that works locally on my desktop. I am trying to figure out how to make it accessible to people at other locations. In other words I guess I'm trying to get it hosted....is that the right term? Jesus I'm so old I sound like my grandpa. Anyway, I've tried uploading my index.html file to both Netlify and GitHub Pages...but when I click the link provided by either service, it gives me a 404. 

Again and to paraphrase, when I double click my index.html on my desktop, it launches it locally and functions as intended. When I try to upload it to Netlify or Git, it appears to be successful, but when I click on the link to view the page, I get a 404. Is there something obvious that I am missing? Something stupid I am doing? Is this an easy fix? I've been talking to ChatGPT and it's like "I got nothing...it should totally be working"

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by tejasraman** [comment, marked as answer]
It’s currently called `index.html.html`… That’ll never work. Name it just `index.html`.
@baysidemeef

**Reply by baysidemeef** [reply]
Haha wait really? Oh my God you're right. I feel dumb. Woof...........thank you

**Reply by tejasraman** [reply]
You’re welcome :)

**Reply by baysidemeef** [reply]
WOW yup that was the problem. I thought if I named a file "index.html" and it's already in HTML format, it wouldn't add the extra ".html" to the file name....guess I was wrong. Thanks again!!!

**Reply by queenofcorgis** [reply]
Nice work @tejasraman and glad you got your answer @baysidemeef - now don't forget to mark it in case another user encounters the same 😄 ✅

**Reply by baysidemeef** [reply]
Oh didn't realize I'm supposed to do that haha...I just clicked "Mark". Thanks again to all for assistance :)
